### ISSUE TITLE
KeyError: 'artist'

### ISSUE THREAD

**Issue by lazka** [quodlibet/quodlibet#2913]
https://sentry.io/share/issue/a98c474ded464f559c52a5bd36f18abc/

```
KeyError: 'artist'
(1 additional frame(s) were not displayed)
...
  File "quodlibet/qltk/songsmenu.py", line 235, in __handle
    if any(ret):
  File "quodlibet/ext/songsmenu/cover_download.py", line 345, in plugin_album
    dialog = CoverArtWindow(songs, manager, Config())
  File "quodlibet/ext/songsmenu/cover_download.py", line 168, in __init__
    self._groups = manager.search_cover(cancellable, songs)
  File "quodlibet/util/cover/manager.py", line 290, in search_cover
    del song['artist']
  File "quodlibet/formats/_audio.py", line 221, in __delitem__
    dict.__delitem__(self, key)

KeyError: 'artist'
```

**Comment by lazka** [comment]
@declension

**Comment by declension** [comment]
Thanks

**Comment by declension** [comment]
Fixed in  16fd3b5
